repose-aggregator/components/services/http-client-service/http-client-service-api/src/main/resources/META-INF/schema/config/http-connection-pool.xsd

```diff
@@ -239,14 +239,22 @@
             </xs:annotation>
         </xs:attribute>
 
-        <xs:attribute name="chunked-encoding" type="xs:boolean" use="optional" default="true">
+        <xs:attribute name="chunked-encoding" type="chunkedEncodingType" use="optional" default="true">
             <xs:annotation>
                 <xs:documentation>
                     <html:p>
-                        Determines whether or not this client should send chunked data when forwarding requests.
+                        Determines whether or not this client should send chunked data when forwarding requests to
+                        the origin service.
                     </html:p>
                     <html:p>
-                        WARNING: Setting this attribute will cause repose to attempt to retrieve the
+                        If set to true or 1, chunked data will always be sent.
+                        If set to false or 0, chunked data will never be sent.
+                        If set to auto, chunked data will only be sent if the original request to Repose was chunked.
+                        Note that the auto setting only has meaning for the default connection pool, as that is the only
```

Comment:
I'd probably also mention that 1 and 0 are deprecated and slated for removal in 9, and then be sure to add that to our 9 timebomb list.